custom_requirements.txt

```diff
@@ -1,4 +1,4 @@
 # SPDX-License-Identifier: MIT
 
 # Write requirements for running your custom commands in tox here:
-PyYAML
+PyYAML; python_version >= '2.7' and python_version != '3.0' and python_version != '3.1' and python_version != '3.2' and python_version != '3.3' and python_version != '3.4'
```

Comment:
Does it also support `or` then this could be `python_version == '2.7' or python_version >= '3.5'` AFAIU.

Reply:
Yes, `or` operator is also supported. Fixed in 8448660.

Reply:
Please squash the commits